common/browser_context_options.go

```diff
@@ -232,3 +232,29 @@ func (w *WaitForEventOptions) Parse(ctx context.Context, optsOrPredicate goja.Va
 
 	return nil
 }
+
+// GrantPermissionsOptions are the options used by the browserContext.grantPermissions
+// API.
+type GrantPermissionsOptions struct {
+	Origin string
+}
+
+// NewGrantPermissionsOptions creates a new instance of GrantPermissionsOptions.
```

Comment:
I don't mind, but just a minor suggestion to have a more idiomatic comment on pkg.go.dev:

```suggestion
// NewGrantPermissionsOptions returns a new GrantPermissionsOptions.
```